sources/assets/exegol/load_supported_setups.sh

```diff
@@ -131,6 +131,33 @@ function deploy_firefox_addons() {
   fi
 }
 
+function deploy_bloodhound_customqueries_replacement() {
+  # Merge all the *.json files in customqueries format from my-resources/.../replacement to replace the Exegol provided one
+  if \
+    [[ -d "$MY_Setup_PATH/bloodhound_customqueries/replacement" ]] && \
+    [[ -n $(find "$MY_Setup_PATH/bloodhound_customqueries/replacement" -type f -name "*.json") ]]; then
+      bqm --verbose --ignore-default --output-path /root/.config/bloodhound/customqueries.json.bqm -i "$MY_Setup_PATH/bloodhound_customqueries/replacement"
```

Comment:
Can you use `~/` instead of `/root/` ?

Reply:
Done. I changed another path in the same file, and unrelated to bloodhound.